Tests/Tests/AppStartEventSpec.swift

```diff
@@ -0,0 +1,26 @@
+import Quick
+import Nimble
+@testable import RInAppMessaging
+import CoreFoundation
```

Comment:
Is this import necessary? Shouldn't that be just `import Foundation`?
(Same question for other files)